alphagov/paas-product-pages
# Update dependencies

## What 

Regular dependency updates.

Contains updates to:
- react & node types
- rollup, sass & postcss
- commonjs and node-resolve rollup plugin which have moved repos

## How to test

- install updated dependencies (`npm install`)
- build the site (`npm run build`)
- cd into `out` folder and serve the page (`npx serve`)
- check page on `localhost:5000` for any issues

There are [expected warnings in CSS compilation](https://github.com/alphagov/govuk-frontend/issues/2238) coming from govuk-frontend
